tests/Agent/UnitTests/CompositeTests/Samplers/GcSamplerNetCoreTests.cs

```diff
@@ -75,75 +75,75 @@ public void SamplerStartsWithoutException()
             sampler.Start();
         }
 
-        [Test]
-        public void SamplerDisposesEventListenerOnException()
-        {
-            var samplerWasStopped = false;
-            var listenerWasDisposed = false;
+        //[Test]
```

Comment:
Do we need to come back to these?

Reply:
I'll include them for .NET framework builds

Reply:
Actually, these tests pass on both Framework and .NET so I'll put them back in.